JavaScript/JavaScriptSDK/Telemetry/Exception.ts

```diff
@@ -77,8 +77,8 @@ module Microsoft.ApplicationInsights.Telemetry {
 
         constructor(exception: Error) {
             super();
-            this.typeName = Common.DataSanitizer.sanitizeString(exception.name || Util.NotSpecified);
-            this.message = Common.DataSanitizer.sanitizeMessage(exception.message || Util.NotSpecified);
+            this.typeName = Common.DataSanitizer.sanitizeString(exception.name) || Util.NotSpecified;
```

Comment:
I trust that Common.DataSanitizer.sanitizeString handles being passed undefined or null?